engine/src/image.cpp

```diff
@@ -951,6 +953,17 @@ Exec_stat MCImage::getprop_legacy(uint4 parid, Properties which, MCExecPoint& ep
             else
                 ep . clear();
         break;
+        // MERG-2014-09-18: [[ ImageMetadata ]] return the image metadata as an array
+    case P_METADATA:
+        if (m_rep != nil)
+        {
+            MCImageMetadata t_metadata;
+            (static_cast<MCCachedImageRep *>(m_rep))->GetMetadata(t_metadata);
+            MCImageGetMetadata(ep, t_metadata);
```

Comment:
OK, I'll take another stab... was sure I had all those classes covered but I guess not.